Terraform/aws.tf

```diff
@@ -0,0 +1,24 @@
+/*terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}*/

```

Comment:
Removing unused code from a codebase is generally considered good practice, rather than simply commenting it out. This is because commented-out code can clutter the codebase and make it more difficult to read and understand. It can also make it harder to see the changes made to the codebase because commented-out code can obscure the changes made to the active code.

Having said that, please remove this block instead of commenting it out